release/_product.sh

```diff
@@ -423,13 +415,42 @@ function warm_up_tomcat {
 
 		cat ../logs/catalina.out
 
-		return 1
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
 	fi
 
 	cat ../logs/catalina.out
 
 	rm -fr ../logs/*
 	rm -fr ../../logs/*
+}
+
+function update_release_info_date {
+	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee
```

Comment:
For consistency, can you change it to `lc_cd "${_PROJECTS_DIR}/liferay-portal-ee"`?

Reply:
Sure, @lucasmiranda0!